src/turbine/runtime/intermediate.py

```diff
@@ -47,7 +47,7 @@ async def write(
             config = {}
 
         if not collection:
-            raise Exception(f"A collection name is required for all resources")
+            raise Exception("A collection name is required for all resources")
```

Comment:
```suggestion
            raise Exception("A collection name is required when calling write()")
```
i feel like this could be a little clearer